### PR TITLE
Add monerium risk assessment

### PR DESCRIPTION
**Pull request by evmKnows**

### CODE REVIEW

File: articles/Monerium/monerium.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+It is important to note that there are residual risks associated with e-money, such as defaulting governments or banks. However, considering that an EMI is likely banking with systemic relevant banks and investing in AAA-rated gov. bonds the remaining risks can be practically disregarded. As a matter of fact, e-money is considerably less risky than the systemic relevant banks which operate in the EU. Firstly, unlike banks that operate under the fractional reserve system, e-money is over-collateralized with 2% of the outstanding issued e-money, as well as a capital contribution of 350k. Secondly, most of the assets are invested into high-quality liquid assets and only the necessary amount is kept at a bank to meet regular withdrawals (as part of a prudent practice), whereby the reserves are also segregated from the EMI. Thirdly, gov. bonds are secured through the ability of the government to tax its citizens, while banks do not have such an option. And lastly, you have diversification benefits by nature of investing in multiple different gov. bonds. All of these factors combined make the case for money that provides a more secure money scheme than demand deposits.
```
Comment (benber86) [inline articles/Monerium/monerium.md:97]: same comment here on over-collateralization, it seems it's a full reserve requirement + additional capital requirements?

File: articles/Monerium/monerium.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
```
Comment (benber86) [inline articles/Monerium/monerium.md:118]: You listed three different options in the previous sections wrt backing, what option did Monerium go for? Did they take additional precautions? (For instance, taking an insurance policy on top of having them invested in low risk assets)

File: articles/Monerium/monerium.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+A quick TL;DR of our findings:
+
+* Monerium offers a EURe stablecoin which can be on- and off-boarded using an [IBAN](https://en.wikipedia.org/wiki/International_Bank_Account_Number) (International Bank Account Number). When registering  with Monerium you get a personal IBAN that you can fund in order to receive EURe on your on-chain wallet. In the same way, you can off-board EURe by funding your on-chain wallet and sending the EURe via the Monerium app to an IBAN. By doing that the EURe is directly burnt from your wallet.
+* Monerium is regulated as an electronic money institution (EMI) for the issuance of e-money.
+* An EMI must meet a number of regulatory requirements, such as over-collateralization with €350k and a 2% buffer based on the outstanding issued amount of e-money, segregating the EMIs funds and reserves by using different custodians and investing the reserves into secure, low-risk assets denominated in euro.
```
[the remark this answers is not in the document]
Comment (evmKnows) [reply articles/Monerium/monerium.md:22]: I'd say over-collateralization is the fitting term because (IIRC) the 350k and 2% is a buffer to cover residual risks related to the safeguarding requirements (can't find the remark rn). Even in the event that for example the HQLAs are sold for a slight loss, and the EMI filed for bankruptcy, the emoney holders have a priority claim.

https://monerium.com/policies/business-terms-of-service/

> According to the Icelandic Bankruptcy Act No. 21/1991, e-money holders enjoy the highest priority claim against the bankruptcy estate in proportion to their ownership. In the event of our insolvency, or if a financial claim is made against us, no creditor or claimant should be able to claim funds held in our Safeguarded Accounts. This is because no other person or institution may have any rights or interest over the funds held in our Safeguarded Accounts such as a lien.

But even without concrete statements from the EMD & MiCA that the 350k and 2% constitutes a buffer to cover residual risks. Because of Article 44(4)

> Upon request by the holder of e-money tokens, the respective issuer must redeem, at any moment and at par value the monetary value of the e-money tokens held to the holders of e-money tokens, either in cash or by credit transfer.

It can be assumed that own capital will be used to fulfill the obligations otherwise the EMI risks the loss of his license, fines and potentially jail time for the people in charge?

The above interpretation is supported by the fact that significant e-money tokens have and additional requirement where it needs 3% instead of 2%

> The percentage referred to in Article 31(1), point (b), shall be set at 3% of the average amount of the reserve assets for issuers of significant asset-referenced tokens.
(^^ applies to Sig. EMT as per Article 52 of MiCA)

The profitability of a stablecoin issuer likely scales faster than the outstanding issued e-money tokens. Thus, the extra 1% only makes sense as an additional buffer to secure the reserves.

File: articles/Monerium/monerium.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+
+>[Article 9(1) of the payment services directive](https://eur-lex.europa.eu/legal-content/EN/ALL/?uri=celex:32007L0064#:~:text=Safeguarding%20requirements-,1,-.%C2%A0%C2%A0%C2%A0The%20Member%20States)
+> 
+>(a) they shall not be commingled at any time with the funds of any natural or legal person other than payment service users on whose behalf the funds are held .. and they shall be deposited in a separate account in a credit institution or invested in secure, liquid low-risk assets as defined by the competent authorities of the home Member State;
+> 
+>(b) they shall be insulated in accordance with national law in the interest of the payment service users against the claims of other creditors of the payment institution, in particular in the event of insolvency;
+>
+>or
+>
+>(c) they shall be covered by an insurance policy or some other comparable guarantee from an insurance company or a credit institution, which does not belong to the same group as the payment institution itself .. payable in the event that the payment institution is unable to meet its financial obligations.
```
[the remark this answers is not in the document]
Comment (evmKnows) [reply articles/Monerium/monerium.md:46]: I suspect no since EMIs usually use pooled accounts. It might be a handful of those, these are ofc insured. Will clarify on this tho

File: articles/Monerium/monerium.md (hunk 1 above)
Comment (evmKnows) [reply articles/Monerium/monerium.md:97]: Okay I think I understand your first remark more now

File: articles/Monerium/monerium.md (hunk 2 above)
Comment (evmKnows) [reply articles/Monerium/monerium.md:118]: Covered in 
### Risk Summary e-money:

> Secondly, most of the assets are invested into high-quality liquid assets and only the necessary amount is kept at a bank to meet regular withdrawals (as part of a prudent practice), whereby the reserves are also segregated from the EMI.

File: articles/Monerium/monerium.md [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+* [The owner](https://etherscan.io/address/0xA176a20E3698bAa36Ea09E4499e407e7dbFdfa0c) of the [EURe contract](https://etherscan.io/address/0x3231cb76718cdef2155fc47b5286d82e6eda273f#writeContract) has the ability to upgrade the controller. (And hence being able to infinitely mint EURe)
+* [The owner](https://etherscan.io/address/0xA176a20E3698bAa36Ea09E4499e407e7dbFdfa0c) of the [EURe contract](https://etherscan.io/address/0x3231cb76718cdef2155fc47b5286d82e6eda273f#writeContract) has the ability to call the `selfdestruct` function, which kills the contract and transfers all EURe to the owner.
+* [A SystemAccount](https://etherscan.io/address/0x882145b1f9764372125861727d7be616c84010ef) (within the controller) is an EOA.
+* [A SystemAccount](https://etherscan.io/address/0x882145b1f9764372125861727d7be616c84010ef) is able to (infinitely) mint EURe.
+* [The controller contract](https://etherscan.io/address/0x914C6a264bC3cC68966C33BBD6630F011E7a47cF) is not verified on etherscan.
```
Comment (chanhosuh) [inline articles/Monerium/monerium.md:139]: Should make sure they verify all contracts on Etherscan.  This is a red flag.

File: articles/Monerium/monerium.md [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+
+Followed by identifying the above, we reached out to the Monerium team to discuss the issues and how they could potentially be resolved. The result of this discussion was satisfactory in the sense that we were promised that all the issues listed above will be taken care of. This includes:
+
+* Replacing the owner of the EURe contract with a multi-sig
+* Keeping the SystemAccount as an EOA (for operational reasons > automating mints/burns), however implementing changes into the controller that would put a limit on how much EURe can be minted. This would greatly reduce the repercussions of a compromised SystemAccount.
```
Comment (chanhosuh) [inline articles/Monerium/monerium.md:144]: ... and there is no way for anyone to check they deploy these changes without a verified controller contract.

File: articles/Monerium/monerium.md (hunk 2 above)
Comment (benber86) [reply articles/Monerium/monerium.md:118]: Ok, I wasn't sure if this you were talking about Monerium specifically here or making a general statement about what EMIs generally do, since the subsection is called "Risk Summary E-money" and talks about general features of EMI/e-money before getting into the specifics into the dedicated "Monerium" section next.

Maybe rephrase this sentence to say this is specifically what Monerium is doing (could also mention that they're not insured) or move/repeat it in the Monerium section (with all the EU legalese in the first section wouldn't be surprised if a lot of readers skip :sweat_smile:

File: articles/Monerium/monerium.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (gislik) [reply articles/Monerium/monerium.md:46]: As mentioned EMIs may safeguard customer funds in 3 places:

1. Custodial bank accounts
2. Safeguarded in HQLA
3. UCITS funds which comply with 1. and 2.

When EMIs are safeguarded in custodial bank accounts, the assets are ringfenced, from other activities of the EMI and from other activities of the bank. This essentially means that all safeguarded assets are held on behalf of the customer and the bank cannot use the capital for lending purposes.

File: articles/Monerium/monerium.md (hunk 1 above)
Comment (gislik) [reply articles/Monerium/monerium.md:97]: See the comment above. Monerium EMI safeguards 100% of its customer funds in addition to providing at least 2% of its own capital to meet its liabilities.

File: articles/Monerium/monerium.md (hunk 2 above)
Comment (gislik) [reply articles/Monerium/monerium.md:118]: All customer funds are safeguarded in segregated customer accounts and they are not commingled with the company's accounts. The law protects against insolvency of the EMI in such a way that holders of EURe have a direct claim on the safeguarded funds which has a priority above everything else, including employee salary payments. The cool thing is that because the EURe is minted on a blockchain, proving the claim is trivial.

File: articles/Monerium/monerium.md (hunk 5 above)
Comment (gislik) [reply articles/Monerium/monerium.md:139]: Yes, we will. 

At the time of deployment, we only verified the EURe token itself. We will verify the rest of the contracts which comprise the token system. Unfortunately, this is not as straightforward as it should be, because of different compiler versions, maximum library dependencies (10) by the Etherscan verifier, etc. We will however find a solution to verifying all the accounts.

File: articles/Monerium/monerium.md (hunk 6 above)
Comment (gislik) [reply articles/Monerium/monerium.md:144]: See comment above 🙌

File: articles/Monerium/monerium.md [hunk 7]
```diff
@@ -0,0 +1,155 @@
...
+* [The equirement to monitor liquidity needs to meet ad-hoc redemption requests.](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52020PC0593#:~:text=Issuers%20of%20significant%20asset%2Dreferenced%20tokens%20shall%20assess%20and%20monitor%20the%20liquidity%20needs%20to%20meet%20redemption%20requests%20or%20the%20exercise%20of%20rights%2C%20as%20referred%20to%20in%20Article%2034%2C%20by%20holders%20of%20asset%2Dreferenced%20tokens.)
+* [The over-collateralization percentage shall be 3% instead of 2%](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52020PC0593#:~:text=The%20percentage%20referred%20to%20in%20Article%C2%A031(1)%2C%20point%20(b)%2C%20shall%20be%20set%20at%203%25%20of%20the%20average%20amount%20of%20the%20reserve%20assets%20for%20issuers%20of%20significant%20asset%2Dreferenced%20tokens.)
+* [A plan should be in place to be able to orderly wind down the company's activities.](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52020PC0593#:~:text=Chapter%206-,Orderly%20wind%2Ddown,-Article%2042%0AOrderly)
+
+### Smart Contracts
```
[the remark this answers is not in the document]
Comment (evmKnows) [reply articles/Monerium/monerium.md]: I'd say this point is covered in "Review / Smart Contracts"

> While we were able to determine that Monerium has an EMI license and thus has to comply with strict safeguarding requirements we still have to emphasize the importance of trust assumptions that apply in this case. Currently, there are no publicly available proof-of-reserves attestations, audits or similar documents.
> ..
> * Publish regular proof-of-reserves attestations from an auditor [or through other ways](https://gov.curve.fi/t/proposal-to-add-ageur-eure-on-ethereum-to-the-gauge-controller/4544/7?u=knows)

**Review by bout3fiddy — APPROVED** [review]
LGTM. Thank you for the work! This is very thorough and neutral.
